.spelling

```diff
@@ -589,6 +589,7 @@ remediated
 remediates
 remediating
 respawns
+rollout
```

Comment:
Add rollout and rollouts to the global list, instead of defining it in multiple file exclusion lists.

Reply:
I can do that, it just wasn't clear to me if we wanted to accept those words as correct spelling across the entire repo.

Reply:
Fixed in next commit.